public_html/siafunds-alpha.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+
+<!--#include file="ssi/head.html" -->
+
+<body>
+	<div class='container'>
+		<div class='page'>
+
+			<!--#include file="ssi/header.html" -->
+
+			<!-- Description title header -->
+			<div class='row'>
+				<div class='col-xs-12'>
+					<div class='content-title'>
+						Siafunds Alpha
+					</div>
+					<div class='content-subtext'>Siafunds and Siac</div>
+				</div>
+			</div>
+
+			<!-- Description paragraphs -->
+			<div class='row'>
+				<div class='col-xs-12'>
+					<div class='content'>
+						The early siafund tool is ready for use.
+						Please be careful when using this tool, incorrect use can cause loss of siafunds.
+						In the coming weeks and months, simpler and more forgiving tools will be created for working with siafunds.
+						<br><br>
+						<b>Step 1: Get your new siafund address.</b><br>
+						The new version of Sia uses addresses with checksums.
+						This means that all of your old addresses will be recognized as invalid, and you need to regenerate them.
+						The checksums will prevent typos from sending coins into oblivion.
+						To get your new address, you will need to use siag again.
+						<b>'siag keyinfo [keyfile]'</b>
+						This will print out your updated address.
+						The updated address will receive all siafunds sent to the old address.
+						<br><br>
+						<b>Step 2: Help the wallet find your siafunds.</b><br>
+						Siac has a command to help you track a siafund wallet.
+						<b>'siac wallet siafunds track [address]'</b>
+						Note that this does not load your private key into the wallet, it merely allows the wallet to follow your siafunds around.
+						You still need to keep your keyfiles.
+						After you have told the wallet to track your siafunds, you will need to restart siad.
+						After restarting siad, all the siafunds owned by the key you are tracking should appear in the wallet.
+						<br><br>
+						<b>Step 3: Spending siafunds.</b><br>
+						Siafund spending is a little different than siacoin spending.
+						Only transactions that make it into the blockchain are considered by the wallet.
+						Because of this, the wallet will sometimes (or frequently) try to spend siafunds that it has already spent.
+						To be safest, you should only make 1 spend at a time, then wait until the transaction appears in a block before making another spend.
```

Comment:
consider rewording -- for a moment I thought this was saying you should only send 1 siafund